# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1530021139 14400
#      Tue Jun 26 09:52:19 2018 -0400
# Node ID ec2492a2af6ab335bab669ecbfaad3e368c1a934
# Parent  b1aefc73af6e70a8e7fcfc47894d9a4f8184fcc5
* mkoctfile.in.cc [CROSS]: Include <fcntl.h> for O_BINARY.

diff --git a/src/mkoctfile.in.cc b/src/mkoctfile.in.cc
--- a/src/mkoctfile.in.cc
+++ b/src/mkoctfile.in.cc
@@ -32,19 +32,20 @@ along with Octave; see the file COPYING.
 #include <algorithm>
 #include <iostream>
 #include <fstream>
 #include <sstream>
 #include <vector>
 #include <cstdlib>
 
 #if defined (CROSS)
+#  include <fcntl.h>
+#  include <stdlib.h>
 #  include <sys/types.h>
 #  include <sys/wait.h>
-#  include <stdlib.h>
 #  include <unistd.h>
 
 #  ifndef OCTAVE_UNUSED
 #    define OCTAVE_UNUSED
 #  endif
 #else
 #  include "mkostemps-wrapper.h"
 #  include "unistd-wrappers.h"
